File: Language/Java/Phase-2/LeapYear.java
// Write a program to check whether a given year is a leap year or not.

// Input: year = 2020
// Output: 2020 is a leap year

import java.util.Scanner;

public class LeapYear {
    public static void leapyr(int n){
        if((n%4 == 0 && n%100 == 0) || n%400 == 0){
            System.out.println("Leap year");
        }
        else{
            System.out.println("Not a Leap year");
        }
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a year: ");
        int n = sc.nextInt();
        leapyr(n);
        sc.close();
    }
}

File: Python/Basic/LeapYear.py
# Write a program to check whether a given year is a leap year or not.

# Input: year = 2020
# Output: 2020 is a leap year

x = int(input("Enter a year: "))
if (x%4 == 0 and x%100 != 0) or x%400 == 0:
    print("Leap year")
else:
    print("Not a leap year")